### DISCUSSION TITLE
Quick Terminal at center - edge handles move instead of resize

### DISCUSSION THREAD

**Discussion by edrozenberg** [ghostty-org/ghostty#3609] [Feature Requests, Ideas] [answered]
Ghostty Version 1.0.0
Build 8469
Commit 4b4d4062

(MacOS)

When using `quick-terminal-position = center`, the resizing handles do not resize the terminal in any direction. They only cause the Quick Terminal to move around the desktop, a move which is not persistent since hiding/showing Quick Term again resets it to the centered position.

Expected behavior: resizing handles allow resizing such that the terminal remains centered, i.e. for ex. resizing down would also cause a matching simultaneous resize up which would maintain the center position. My understanding is the Quick Terminal is not intended to be moved around, so the resizing handles should be affecting the window size, not the window position.

https://github.com/user-attachments/assets/bc766ac6-3eeb-45fe-8d2c-b8a1808d113d

**Comment by injust** [comment]
Instead of the resize handles moving the quick terminal around, they do nothing for me.

I'm using Ghostty 1.1.3 on macOS Sequoia 15.5.

**Comment by mitchellh** [comment, marked as answer]
#8398
